test/screenshot/fixture.scss

```diff
@@ -52,13 +51,8 @@ $test-button-columns: 2;
   margin: 0 #{$test-button-cell-margin} #{$test-button-cell-margin} 0;
 
   // Ensure that shadows from adjacent components don't overlap
-  padding: $test-ruler-granularity;
+  padding: $test-button-cell-padding;
 
   // Ruler grid pattern
-  // https://stackoverflow.com/a/32861765/467582
-  background-image:
-    linear-gradient(to right, #{$test-ruler-color} 1px, transparent 1.01px),  // fraction for IE 11
-    linear-gradient(to bottom, #{$test-ruler-color} 1px, transparent 1.01px); // fraction for IE 11
-
-  background-size: #{$test-ruler-granularity} #{$test-ruler-granularity};
+  background-image: url("images/ruler-grid.png");
```

Comment:
Inline the base64 image so you can drop the b64 webpack dependency

Reply:
Done - good call 😄